snakesist/exist_client.py

```diff
@@ -403,18 +403,17 @@ def query(self, query_expression: str) -> etree._Element:
         )
         return etree.fromstring(response_string, parser=self.parser)
 
-    # TODO? rename
-    def retrieve_resources(self, xpath: str) -> List[NodeResource]:
+    def xpath(self, expression: str) -> List[NodeResource]:
         """
         Retrieve a set of resources from the database using
```

Comment:
if this is restricted to the currently set `root_collection`, it should be mentioned here.

Reply:
true, it is and it should me mentioned, of course.